city_scrapers/spiders/middlesex_co.py

```diff
@@ -0,0 +1,205 @@
+        "name": "middlesex_co_tb",
+        "agency": "TAX BOARD",
+        "committee_id": 38,
+    },
+    {
+        "class_name": "MiddlesexCoICSpider",
+        "name": "middlesex_co_ic",
+        "agency": "Middlesex County Insurance Commission",
+        "committee_id": 52,
+    },
+    {
+        "class_name": "MiddlesexCoWMPAPHSpider",
+        "name": "middlesex_co_wmpaph",
+        "agency": "Wastewater Management Plan Amendment Public Hearings",
+        "committee_id": 47,
+    },
+    {
+        "class_name": "MiddlesexCoWOSCSpider",
+        "name": "middlesex_co_wosc",
+        "agency": "WDB One Stop Committee",
+        "committee_id": 48,
+    },
+    {
+        "class_name": "MiddlesexCoSWACSpider",
+        "name": "middlesex_co_swac",
+        "agency": "Solid Waste Advisory Council (SWAC)",
+        "committee_id": 36,
+    },
+    {
+        "class_name": "MiddlesexCoSWMPHSpider",
+        "name": "middlesex_co_swmph",
+        "agency": "Solid Waste Management Public Hearings",
+        "committee_id": 57,
+    },
+    {
+        "class_name": "MiddlesexCoCoCAaMCSpider",
+        "name": "middlesex_co_cocaamc",
+        "agency": "Commission on Child Abuse and Missing Children",
+        "committee_id": 5,
+    },
+    {
+        "class_name": "MiddlesexCoCfCSSpider",
+        "name": "middlesex_co_cfcs",
+        "agency": "Council for Children's Services",
+        "committee_id": 7,
+    },
+    {
+        "class_name": "MiddlesexCoHTFABSpider",
+        "name": "middlesex_co_htfab",
+        "agency": "Homeless Trust Fund Advisory Board",
+        "committee_id": 55,
+    },
+    {
+        "class_name": "MiddlesexCoHSACSpider",
+        "name": "middlesex_co_hsac",
+        "agency": "Human Services Advisory Council",
+        "committee_id": 15,
+    },
+    {
+        "class_name": "MiddlesexCoLACfAaDASpider",
+        "name": "middlesex_co_lacfaada",
+        "agency": "Local Advisory Commission for Alcoholism and Drug Abuse",
+        "committee_id": 18,
+    },
+    {
+        "class_name": "MiddlesexCoMHBSpider",
+        "name": "middlesex_co_mhb",
+        "agency": "Mental Health Board",
+        "committee_id": 25,
+    },
+    {
+        "class_name": "MiddlesexCoHCoCCSpider",
+        "name": "middlesex_co_hcocc",
+        "agency": "Middlesex County Housing Continuum of Care Committee",
+        "committee_id": 54,
+    },
+    {
+        "class_name": "MiddlesexCoOoA&DSSpider",
+        "name": "middlesex_co_ooa&ds",
+        "agency": "Middlesex County Office of Aging & Disabled Services",
+        "committee_id": 59,
+    },
+    {
+        "class_name": "MiddlesexCoRWACSpider",
+        "name": "middlesex_co_rwac",
+        "agency": "Ryan White Advisory Council",
+        "committee_id": 53,
+    },
+    {
+        "class_name": "MiddlesexCoVASpider",
+        "name": "middlesex_co_va",
+        "agency": "Veterans' Advisory",
+        "committee_id": 40,
+    },
+]
```

Comment:
Hi @SimmonsRitchie The spider name must start with `newbnj_`
<img width="608" alt="Screenshot 2024-03-01 at 15 03 34" src="https://github.com/City-Bureau/city-scrapers-new-brunswick/assets/1421838/2c283f04-3aee-415b-a334-2d6c530dce57">

Reply:
> newbnj_

Ah, good catch! My bad. Sloppy on my part. Just push a commit that should fix that across the spiders and tests.

Reply:
Whoops, I see I inadvertently caused a style issue that led CI to fail. Now resolved, @LienDang. This PR is ready for your final sign off.